src/main/java/com/openshift/jenkins/plugins/freestyle/BaseStep.java

```diff
@@ -147,13 +149,12 @@ protected boolean runOcCommand(final AbstractBuild build,
         if (c == null) { // if null, we assume the cluster is running the
                          // Jenkins node.
             server = ClusterConfig.getHostClusterApiServerUrl();
-            verboseOptions.add("--certificate-authority="
-                    + SERVICE_ACCOUNT_CA_PATH);
+            selectedCAPath = SERVICE_ACCOUNT_CA_PATH;
             caContent = null;
         } else {
             server = c.getServerUrl();
-            if (c.isSkipTlsVerify()) {
-                options.add("--insecure-skip-tls-verify");
+            if (c.isSkipTLSVerify()) {
```

Comment:
OK, I see where you've removed the skip tls / ca cert params that were being passed in, but whose values were always false and "", but I'm still concerned about the advance arguments.  

I realize now my prior comment about parsing them was not entirely sufficient.  It is not just that we need to parse for the skip tls and certs arg specifically.  In case of the "raw" command, I realize now when looking at the jelly files for the config under src/main/resources that the advance args from base step apply specifically to the  options of `oc` that start with `--` that work across all commands.  The raw step has a separate field that applies to the command being run. And as such, the advance args should appear prior to the invocation of the command in question.

The code and its currently lumping of advanced args into userArgs then muddles things when we order things in OcAction, especially in the case of rsh/exec.

We may have to add new config fields.

A blue jeans chat where we look at the UI panels, review the various oc options, etc. might be warranted wrt how we move forward.

Now, the create/delete are less clear

Reply:
At worst, maintaining a specific set of options might be needed (though certainly it is desirable to avoid as options will be added, etc. over time)